core/src/main/java/ai/timefold/solver/core/impl/score/stream/bavet/BavetConstraintFactory.java

```diff
@@ -92,8 +95,8 @@ public <Stream_ extends BavetAbstractConstraintStream<Solution_>> Stream_ share(
     // from
     // ************************************************************************
 
-    @Override
-    public <A> @NonNull UniConstraintStream<A> forEach(@NonNull Class<A> sourceClass) {
+    public <A> @NonNull UniConstraintStream<A> forEachUnfilteringSpecified(@NonNull Class<A> sourceClass,
```

Comment:
IMO a better name is in orther. 

`forEachWithFilters`?
`forEachUnfilteredWithIncludes`?